docs/forms-in-wagtail.md

```diff
@@ -0,0 +1,40 @@
+# Including forms into Wagtail page context
+
+Wagtail provides a FormBuilder module, but it cannot be used with subclasses of Page, like our CFGOVPage. For our purposes there is `AbstractFormBlock`, a subclass of StructBlock that implements methods to process a request. If a developer wishes to add a module that includes a form, they only need to follow a few steps in order to get it handled properly:
+
+1. Create the form.
```

Comment:
Coming out of our forms-that-store vs forms-that-don't conversation, I think I'd like to see a parenthetical note here or something that the form can be a a Form or ModelForm subclass backed by a model. But that's nothing that should hold this PR up any longer.

Thinking forward, it might even be worth including a Model and ModelForm in the example, even if that's a known Django pattern, just to emphasize how doable it is.